### PR TITLE
[fix #3163] remove phone command

### LINKED ISSUE #3163: Remove phone command from Console, consider update from previous version and infrustructure
### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature

[comment]: # (Describe the feature you would like, or briefly summarise the bug and what you did, what you expected to happen, and what actually happens. Sections below)
*Summary*: 
1. Remove phone command in Console because we removed it in the profile and will not support in beta. Currently, text message to confirm number is not received by users in several countries and many users complain. So, it's better to remove it even before onboarding is re-designed. 
2. Consider upgrade from older version (where phone number is available for some users) - do we need to clean up db or anything else?
3. Suspend account/service used to send text messages, any other servers involved?

#### Expected behavior
[comment]: # (Describe what you expected to happen.)
1. No phone command in Console
2. If upgrade from older version with account that provided phone number - can use current version

### Reproduction
[comment]: # (Describe how we can replicate the bug step by step.)

- Open Status
- Open Console

### PR DESCRIPTION
**Pull request by yenda**
Removing the phone command

### CODE REVIEW

**Review by janherich — APPROVED** [review]

**Comment by yenda** [comment]
Branch: PR-3169
Android: https://i.diawi.com/DB2wnG
iOS: https://i.diawi.com/vnBUcN

File: src/status_im/translations/ar.cljs [hunk 1]
```diff
@@ -29,11 +29,11 @@
    :suggestions-requests                  "الطلبات"
    :suggestions-commands                  "الأوامر"
 
-   ;sync
+                                        ;sync
```
Comment (oskarth) [inline src/status_im/translations/ar.cljs]: @yenda Can we please not do format changes in PRs in future? It makes it hard to review the actual diff.

File: src/status_im/translations/da.cljs [hunk 2]
```diff
@@ -98,28 +98,17 @@
...
-   :phone-number-required                 "Klik her for at indtaste dit telefonnummer, så vil jeg finde dine venner"
    :shake-your-phone                      "Fundet en bug eller har du et foreslag. Blot ryst din telefon!"
    :intro-status                          "Chat med mig for at opsætte din konto og ændre dine indstillinger!!"
    :intro-message1                        "Velkommen til Status\n Tryk på denne meddelelse for at angive din adgangskode og komme i gang!"
    :account-generation-message            "Vent et øjeblik, jeg er nødt til at gøre nogle skøre beregninger for at generere din konto!"
    :move-to-internal-failure-message      "Vi skal flyyte nogle vigtige filer fra din eksterne til din interne hukommelse. For at gøre dette har vi brug for din tilladelse. Vi vil ikke bruge din eksterne hukkomelse i kommende versioner."
    :debug-enabled                         "Debug server has been launched! You can now execute *status-dev-cli scan* to find the server from your computer on the same network."
 
-   ;phone types
```
Comment (oskarth) [inline src/status_im/translations/da.cljs]: Maybe we don't need to touch the translations at all? Don't see problem with having them there tbh, especially if we bring it back in future. Easier to just not render data and leave it be maybe?

Comment (flexsurfer) [reply src/status_im/translations/da.cljs]: agree with @oskarth let's do not remove translations

File: src/status_im/translations/ar.cljs (hunk 1 above)
Comment (yenda) [reply src/status_im/translations/ar.cljs]: ok I moved the reformatting in #3204

File: src/status_im/translations/da.cljs (hunk 2 above)
Comment (yenda) [reply src/status_im/translations/da.cljs]: removed and moved to #3204 I told you there would be translations removed in phone command @flexsurfer :D

**Comment by oskarth** [comment]
@yenda Did you try upgrading from 0.9.12? :D Also conflicts

**Comment by yenda** [comment]
@oskarth no but I based the migration on the location migration should be good. But that should be tested for sure.
I'll handle the conflict later it's irrelevant indentation change anyway

**Comment by oskarth** [comment]
@yenda Ok, but Probot might disagree with this! In which case conflicts have to be resolved :P Moving to TO TEST.

**Comment by janherich** [comment]
@yenda please resolve conflict

**Comment by annadanchenko** [comment]
After providing password, when account is created: ```Missing value for property 'account.whisper-identity'```
https://app.testfairy.com/projects/4803622-status/builds/7648915/sessions/1/?accessToken=9OR0SYKh/bxi5r5p8uL2GhYKHVE
On develop build issue does not happen

both Android and ios
from log:
```
NAME: "Error"
MESSAGE: Missing value for property 'account.whisper-identity'

create@[native code]
cljs$core$IFn$_invoke$arity$4@index.android.bundle:12:6076163
index.android.bundle:12:6086342
write@[native code]
write@index.android.bundle:12:6074667
save@index.android.bundle:12:6086517
save@index.android.bundle:12:6087172
G__39147_39149@index.android.bundle:12:6361980
index.android.bundle:12:4648968
invoke_interceptor_fn@index.android.bundle:12:4604967
invoke_interceptors@index.android.bundle:12:4605268
execute@index.android.bundle:12:4606886
handle@index.android.bundle:12:4617395
re_frame$router$IEventQueue$_process_1st_event_in_queue$arity$1@index.android.bundle:12:4644664
re_frame$router$IEventQueue$_run_queue$arity$1@index.android.bundle:12:4628046
index.android.bundle:12:4631226
re_frame$router$IEventQueue$_fsm_trigger$arity$3@index.android.bundle:12:4635437
index.android.bundle:12:4629348
index.android.bundle:32:1772
b@index.android.bundle:32:628
w@index.android.bundle:32:992
callImmediates@index.android.bundle:32:3173
value@index.android.bundle:27:2778
index.android.bundle:27:1360
value@index.android.bundle:27:2606
value@index.android.bundle:27:1330
[native code]
value@index.android.bundle:27:1265
[native code]
```

**Comment by yenda** [comment]
@annadanchenko Branch: PR-3169
Android: https://i.diawi.com/BNqKvC
iOS: https://i.diawi.com/C3VRrX

**Comment by annadanchenko** [comment]
@yenda issue with `Missing value for property 'account.whisper-identity'` is fixed in new build
Now new one is shown when sending message in 1:1 chat to Jarrad or joining public chat (I haven't checked it before):
TF (00:35) https://app.testfairy.com/projects/4803622-status/builds/7656779/sessions/1/?accessToken=prIOURdbzNsyOYLeb9qjjov5hpI

``` 00:34 I/ReactNativeJS: NAME: "TypeError"
00:34 I/ReactNativeJS: MESSAGE: null is not an object (evaluating 'e.shh')
00:34 I/ReactNativeJS: 
00:34 I/ReactNativeJS: add_sym_key_from_password@index.android.bundle:12:6113387
00:34 I/ReactNativeJS: cljs$core$IFn$_invoke$arity$4@index.android.bundle:12:6114875
00:34 I/ReactNativeJS: cljs$core$IFn$_invoke$arity$3@index.android.bundle:12:6114044
00:34 I/ReactNativeJS: prepare_message@index.android.bundle:12:6125311
00:34 I/ReactNativeJS: index.android.bundle:12:6136092
00:34 I/ReactNativeJS: s@index.android.bundle:12:6131081
00:34 I/ReactNativeJS: run_state_machine@index.android.bundle:12:5002879
00:34 I/ReactNativeJS: run_state_machine_wrapped@index.android.bundle:12:5003031
00:34 I/ReactNativeJS: index.android.bundle:12:5003479
00:34 I/ReactNativeJS: index.android.bundle:12:4989188
00:34 I/ReactNativeJS: process_messages@index.android.bundle:12:4982698
00:34 I/ReactNativeJS: index.android.bundle:32:1772
00:34 I/ReactNativeJS: b@index.android.bundle:32:628
00:34 I/ReactNativeJS: w@index.android.bundle:32:992
00:34 I/ReactNativeJS: callImmediates@index.android.bundle:32:3173
00:34 I/ReactNativeJS: value@index.android.bundle:27:2778
00:34 I/ReactNativeJS: index.android.bundle:27:1360
00:34 I/ReactNativeJS: value@index.android.bundle:27:2606
00:34 I/ReactNativeJS: value@index.android.bundle:27:1330
00:34 I/ReactNativeJS: [native code]
00:34 I/ReactNativeJS: value@index.android.bundle:27:938
00:34 I/ReactNativeJS: [native code]```

**Comment by annadanchenko** [comment]
found issues on previos builds are fixed
can upgrade ios with 0.9.13 release and phone set 
can upgrade android (without phone number) as there is known issue that it can't be set on 0.9.13
both can chat and send messages in 1:1 and group and public chats
strange thing on upgraded iOs: in 1:1 chat there is always 1 unread counter even if all is read - will investigate on develop

**Pull request merged**